lambdas/service/app.py

```diff
@@ -685,6 +711,9 @@ def file_url(uuid, version):
 
 
 def _file_url(uuid, **kwargs):
-    return (config.service_endpoint() + '/dss/files' +
-            '/' + urllib.parse.quote(uuid, safe='') +
-            '?' + urllib.parse.urlencode(kwargs))
+    protocol = app.current_request.headers.get('x-forwarded-proto', 'http')
```

Comment:
Remind me to talk to you about refactoring thins.

Reply:
Done!